recipes/c-ares/all/conanfile.py

```diff
@@ -62,6 +62,7 @@ def generate(self):
         tc.variables["CARES_BUILD_TESTS"] = False
         tc.variables["CARES_MSVC_STATIC_RUNTIME"] = False
         tc.variables["CARES_BUILD_TOOLS"] = self.options.tools
+        tc.blocks["rpath"].skip_rpath=True
```

Comment:
No, don't do that. It makes shared lib non-relocatable. I can cross-build c-ares on macOS, please open an issue.